app/models/order.rb

```diff
@@ -1,4 +1,18 @@
 class Order < ApplicationRecord
   belongs_to :user
   has_many :order_details, dependent: :destroy
-end
+
+  def total_price
+    order_detail.sum do |order_detail|
+      order_detail.sub_total_price
+    end
```

Comment:
こういう書き方ができます！

```suggestion
    order_details.sum(&:sub_total_price)
```

この書き方は大体以下のことができます
- sumやeach, mapみたいな、複数要素をひとつひとつ処理する系のメソッドに対して使える
- 今回みたいにorder_detailsが対象だとすると、order_detailの中にあるメソッドを各要素に処理できる

Reply:
https://github.com/thimon21/railsthimon/pull/10/commits/0f66be621a874b83e546e25da17055d284ca350a
上記対応いたしました。